src/css/App.css

```diff
@@ -34,8 +34,10 @@ Scales the contents to be centered, also based on the minimum of the user's disp
 
 
 /* overriding bootstrap's container-fluid to include Top/Bottom padding */
-.container-fluid{
-  padding: 40px 40px 40px 40px;
+@media (min-width: 768px) {
```

Comment:
I agree with this. I wasn't sure if Jamie wanted it to have that padding on mobile as well. I like it with no padding on mobile.

Reply:
Ah I intended to ask you about that! I'll leave a comment on the trello board about it